client/pages/projects/projects.js

```diff
@@ -11,10 +11,9 @@ const page = Vue.createApp({
   template: `
       <div class="page">
         <header-component></header-component>
+        <project-component></project-component>
         <main-section></main-section>
-        <footer-component></footer-component>
```

Comment:
I don't know why you removing calling the footer from the template of  projects,js file even you are already importing the component

Reply:
It's strange, let me check

Reply:
Yes, by mistake, I added project-component instead of footer-component. Good catch. Thanks